app/src/main/java/com/example/myapplication/mvp/view/CalculatorView.kt

```diff
@@ -14,4 +16,16 @@ class CalculatorView(activity: Activity) : ActivityView(activity) {
     fun setResult(result: String) {
         activity?.text_result?.text = result
     }
+
+    fun showToastError( text : String ) {
+        Toast.makeText(activity,text,Toast.LENGTH_SHORT).show();
+    }
+
+    fun diseablePoint(){
+        activity?.button_point?.isClickable = false
+    }
+
+    fun activePoint(){
```

Comment:
add space before curly braces to comply with excalibur codestyle

Reply:
done